docs/data/date-pickers/date-picker/CustomInput.tsx

```diff
@@ -1,27 +1,35 @@
 import * as React from 'react';
-import dayjs, { Dayjs } from 'dayjs';
+import dayjs from 'dayjs';
 import Box from '@mui/material/Box';
+import { TextFieldProps } from '@mui/material/TextField';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { Unstable_NextDatePicker as NextDatePicker } from '@mui/x-date-pickers/NextDatePicker';
 
-export default function CustomInput() {
-  const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07'));
+type BrowserInputProps = TextFieldProps & {
+  ownerState?: any;
+};
+
+const BrowserInput = (props: BrowserInputProps) => {
```

Comment:
I'm in favor of removing this demo in the future, in favor of detailed sections in the *Custom sub-components* page.